api/helpers/urlToBlob.js

```diff
@@ -0,0 +1,32 @@
+const https =require("https");
```

Comment:
Could you run your autoformatter on this file?

Reply:
yes! now is working properly on my computer! thanks for remembering me